docs/variants/dell_optiplex/building-manual.md

```diff
@@ -1,53 +1,122 @@
 # Dell OptiPlex 7010/9010 Dasharo - building manual
 
-## Building coreboot
+## Release build
 
 To build coreboot image, follow the steps below:
 
 1. Clone the coreboot repository:
 
     ```bash
-    git clone https://review.coreboot.org/coreboot.git
+    git clone https://github.com/dasharo/coreboot.git
     ```
 
-2. Get the submodules:
-
     ```bash
     cd coreboot
-    git submodule update --init --recursive --checkout
     ```
 
-3. Checkout Dasharo branch for OptiPlex 7010/9010 (replace vX.Y.Z with valid
-   version):
+    Replace vX.Y.Z with valid version:
 
     ```bash
-    git remote add dasharo https://github.com/dasharo/coreboot.git
-    git fetch dasharo
     git checkout dell_optiplex_9010_vX.Y.Z
     ```
 
-4. Start docker container:
+    Checkout subdmodules:
+
+    ```bash
+    git submodule update --init --recursive --checkout
+    ```
+
+2. Start docker container:
```

Comment:
Whether all points should not be declared as `1`? If the text formatting is correct, these `1` should be converted into consecutive numbers.

Reply:
Fixed: https://github.com/Dasharo/docs/pull/139/commits/1040d116a5113f6ed8ec300850b2b1b8441499f4